coop_shift/model/shift_registration.py

```diff
@@ -109,7 +109,7 @@ class ShiftRegistration(models.Model):
     #     'This partner is already registered on this Shift !'),
     # ]
 
-    @api.one
+    @api.multi
     def button_reg_close(self):
```

Comment:
When dealing with actions it's best to add a `self.ensure_one()` in the first line of the method.
It might seem useless, but it does actually make the code much more readable.